### PR TITLE
Item detail page layout improvements continued form #611

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

**Comment by grafixeyehero** [comment]
Before
desktop layout
Annotation d2020-01-14 173513129 k2djpg
mobile layout
Annotation dd2d020-01-14 173513129 k2djpg

After
mobile layout
Annotation 2020-01-14 17351329
Annotation 2020-01-14 173513129 k2djpg

desktop layout
Annotation 2020-01-14 1735329

File: src/controllers/itemdetailpage.js [hunk 1]
```diff
@@ -519,11 +520,19 @@ define(["loading", "appRouter", "layoutManager", "userSettings", "connectionMana
             itemBackdropElement.style.backgroundImage = "";
         }
 
+        if ("Person" === item.Type) {
+            itemBackdropElement.classList.add("personBackdropimg-hideondesktop");
+            itemBackdropElement.classList.add("personBackdrop");
```
Comment (JustAMan) [inline src/controllers/itemdetailpage.js]: why cannot you merge two CSS rules into just `personBackdrop`?

Comment (grafixeyehero) [reply src/controllers/itemdetailpage.js]: that will do thanks https://github.com/jellyfin/jellyfin-web/pull/694/commits/89b28d76f58af92c3e42c717b6f34ae593544b30

**Comment by grafixeyehero** [comment]
update 
Annotation 2020-01-17 180025

**Comment by grafixeyehero** [comment]
update https://github.com/jellyfin/jellyfin-web/pull/694/commits/e43438fca8fca037a530b8707bd9f4bceeefb024
header tab scroll
header-tab-scroll

**Comment by MrTimscampi** [comment]
As mentioned on Matrix, the buttons in mainDetailsButton seem to be having different width, making them look misaligned.

The layout of the Video, Audio and Subtitle selectors can also look wrong when a track has a name longer than the field itself (The field gets resized and eventually pushed to the next line).

**Comment by grafixeyehero** [comment]
> 
> 
> As mentioned on Matrix, the buttons in mainDetailsButton seem to be having different width, making them look misaligned.
> 
this happen with button title  
Annotation 2020-01-30 185347

**Comment by grafixeyehero** [comment]
> 
> The layout of the Video, Audio and Subtitle selectors can also look wrong when a track has a name longer than the field itself (The field gets resized and eventually pushed to the next line).
i did't see your issue here what it cause the problem?
Annotation 2020-01-30 192914

**Pull request merged**